aiidalab_widgets_base/databases.py

```diff
@@ -141,7 +141,7 @@ def __init__(
             **kwargs,
     ) -> None:
         providers = OptimadeQueryProviderWidget(embedded=embedded)
-        filters = OptimadeQueryFilterWidget()
+        filters = OptimadeQueryFilterWidget(embedded=embedded)
```

Comment:
What would happen if the `OptimadeQueryFilterWidget` constructor does not support that argument? Would it just silently ignore it?

Reply:
No idea, honestly. I tried following the class hierarchy, but without luck of finding out what might happen. I'd prefer setting a minimum version of `optimade-client` in `setup.json`... Poking @yakutovicha.

Reply:
See [this line](https://github.com/aiidalab/aiidalab-widgets-base/blob/1f1970f678bddca6d60b9f7a01546acc9e1a5612/setup.json#L22).

Reply:
Testing this, it seems the older `optimade-client` versions don't care about the extra parameter. I have now added a note about this, naming the version of the `optimade-client` where this parameter was added to `OptimadeQueryFilterWidget`.